.circleci/config.yml

```diff
@@ -9,80 +9,31 @@ sensu_go_build_env: &sensu_go_build_env
   #### expecting it in the form of
   ####   /go/src/github.com/circleci/go-tool
   ####   /go/src/bitbucket.org/circleci/go-tool
-  environment:
-    # Hard-coding this for now.
-    - SENSU_BUILD_ITERATION: 1
-    - SENSU_RELEASE_REPO: /go/sensu-release
-    - SECURE_VARIABLES_PLAINTEXT_FILE: /go/build-tmp/secure-variables.plain
-    - BUILD_TMP: /go/build-tmp
-
   working_directory: /go/src/github.com/sensu/sensu-go
   docker:
     - image: circleci/golang:1.11.4
 
 jobs:
   build:
```

Comment:
It might be a stupid question but why do we still need the build job? Could we just add the `checkout` step to the test job?

Reply:
I took a stroll down memory lane and perhaps it can be done in a single job https://github.com/sensu/sensu-go/blob/9925b14b72ca03565e577d0ba67156685ede22d2/.circleci/config.yml

Reply:
I'm attempting this now. I read the docs that noted `runs not using Workflows must have a "build" job as entry point` but didn't realize we were using workflows already.